.github/workflows/manageiq_cross_repo.yaml

```diff
@@ -0,0 +1,52 @@
+name: ManageIQ Cross Repo Workflow
+
+on:
+  workflow_call:
+    inputs:
+      test_repo:
```

Comment:
Super minor, but I noticed the "common" thing for workflows is to use dash case (see `fail-fast`, `runs-on`, etc).  Wonder if we should do the same here for consistency.